.tx/config

```diff
@@ -5,14 +5,23 @@ host = https://www.transifex.com
 file_filter = saleor/dashboard/locale/<lang>/LC_MESSAGES/django.po
 source_file = saleor/dashboard/locale/en/LC_MESSAGES/django.po
 source_lang = en
+minimum_perc = 1
+trans.zh-Hans = saleor/dashboard/locale/zh_Hans/LC_MESSAGES/django.po
+trans.zh-Hant = saleor/dashboard/locale/zh_Hant/LC_MESSAGES/django.po
```

Comment:
I don't understand why only zh-hans/hanz are added here as a special case?

Reply:
They have a dash in their name that is translated to an underscore on our side.